rpm/btrfs-balancer.spec

```diff
@@ -17,15 +21,27 @@ Requires:	libkeepalive-glib-tools >= 1.3.3
 %setup -q -n %{name}-%{version}
 
 %build
-
+%qmake5
 make %{?jobs:-j%jobs}
 
 %install
 rm -rf %{buildroot}
-%make_install
+%qmake5_install
 
 %files
 %defattr(-,root,root,-)
+/etc/dbus-1/system.d/*
+/usr/share/dbus-1/system-services/*
 /lib/systemd/system/*
 %{_sbindir}/*
 
+%package sbj-config
```

Comment:
That's OK. I can move it out. Who handles the rest? What will provide the config file then?